examples/jsm/misc/GPUComputationRenderer.d.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  WebGLRenderer,
+  RenderTarget,
+  Texture,
+  DataTexture,
+  Material,
+  ShaderMaterial
+} from '../../../src/Three';
+
+export interface Variable {
+  name: string;
+  initialValueTexture: Texture;
+  material: Material;
+  dependencies: Variable[];
+  renderTargets: RenderTarget[];
+  wrapS: number;
+  wrapT: number;
+  minFilter: number;
+  magFilter: number;
+}
+
+export class GPUComputationRenderer {
+  constructor(sizeX: number, sizeY: number, renderer: WebGLRenderer);
+
+  addVariable(variableName: string, computeFragmentShader: string, initialValueTexture: Texture): Variable;
+  setVariableDependencies(variable: Variable, dependencies: Variable[] | null): void;
+
+  init(void): string | null;
```

Comment:
`void` is not necessary if the method has no parameter.

Reply:
Done.